Sources/GRPCReflectionService/Documentation.docc/ReflectionServiceTutorial.md

```diff
@@ -70,6 +70,25 @@ You can use Swift Package Manager [resources][swiftpm-resources] to add the gene
 In our example the reflection data is written into the "Generated" directory within the target
 so we include the `.copy("Generated")` rule in our target's resource list.
 
+#### SPM Plugin
+
+Reflection data can also be generated via the SPM plugin by including `"reflectionData": true` in `grpc-swift-config.json`. This will generate the same data as running `protoc` above. The generated data will be in your `.build` folder (or `DerivedData` if running from Xcode) and included in the build process. More about [spm-plugin][spm-plugin] can be found here.
```

Comment:
> The generated data will be in your `.build` folder (or `DerivedData` if running from Xcode) and included in the build process.

We should avoid wording it like this: the location isn't stable and users shouldn't search out these files. Instead we should tell them how to access them via the `Bundle`.

Reply:
Updated docs to refer to Bundle